### PR TITLE
[walletconnect-v2.0.4-alpha.2] : Feature - Connect First ChainId on WalletConnect session request

### PR DESCRIPTION
**Pull request by mahmud-bn**
### Description

PR in response to Issue #1076
### Checklist
- [ ] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [ ] The box that allows repo maintainers to update this PR is checked
- [ ] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/walletconnect/src/index.ts [hunk 1]
```diff
@@ -129,7 +130,7 @@ function walletConnect(options?: WalletConnectOptions): WalletInit {
                   // Check if connection is already established
                   if (!this.connector.connected) {
                     // create new session
-                    this.connector.createSession().then(() => {
+                    this.connector.createSession(connectFirstChainId ? {chainId: parseInt(chains[0].id, 16)} : undefined).then(() => {
```
Comment (taylorjdawson) [inline packages/walletconnect/src/index.ts:133]: @mahmud-bn Just double checking here. Do we need to add some check to make sure that `chains` and `chains[0]` are defined? 
For instance:
```suggestion
                    this.connector.createSession(connectFirstChainId && chains && chains[0] ? {chainId: parseInt(chains[0].id, 16)} : undefined).then(() => {
```

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
Can we bump walletconnect in the demo package.json as well to `2.0.4-alpha.2`

File: packages/demo/src/App.svelte [hunk 2]
```diff
@@ -114,38 +114,7 @@
       gnosis,
       dcent
     ],
-    chains: [
```
Comment (taylorjdawson) [inline packages/demo/src/App.svelte:117]: @mahmud-bn Is there a reason we want to get rid of these chains?

File: packages/walletconnect/src/index.ts (hunk 1 above)
Comment (taylorjdawson) [reply packages/walletconnect/src/index.ts:133]: @mahmud-bn Awesome! Just wanted to double check

File: packages/demo/src/App.svelte (hunk 2 above)
Comment (mahmud-bn) [reply packages/demo/src/App.svelte:117]: Oh! Fixed that. I was just re-checking the chain error.

**Review by taylorjdawson — APPROVED** [review]

**Review by aaronbarnardsound — APPROVED** [review]

**Pull request merged**